UPGRADE.md

```diff
@@ -2,6 +2,12 @@
 
 ## dev-develop
 
+### Commands changed
+
+The sulu specific commands where changed instead depending on the deprecated `ContainerAwareCommand` they are replaced
```

Comment:
The Sulu commands were refactored to use the `Command` class instead of the `ContainerAwareCommand` class, because some services are private and cannot be used anymore without using dependency injection. If you have overridden a command before, you have to specify the constructor arguments correctly now.